spectool/README.md

```diff
@@ -0,0 +1,45 @@
+# Intro
+
+The spectools utilies perform conversions between an EBML Schema (an XML representation of the definition of an EBML Document Type) and other human-readable XML formats or library files.
+
+The utilities include:
+
+- data2lib
+- data2lib2
+- data2spec
+- spec2data
+- tabler
+
+# Installation
+
+Building spectools is dependant on coremake. To build for make run these commands from ~/foundation-source.
+
+```
+cd corec/tools/coremake
+make
+sudo make install
```

Comment:
No, you cannot. There's a hard-coded path lookup for `/usr/local/share/coremake` when reading the `.build` files. Therefore you have to install it once or use symlink trickery.